app/assets/javascripts/angular/directives/frame.js.erb

```diff
@@ -17,10 +17,7 @@ angular.module("Prometheus.directives").directive('inlineFrame', ["$sce", "Varia
         return WidgetHeightCalculator(element.find(".js_widget_wrapper")[0], scope.aspectRatio);
       }
 
-      scope.refreshFrame = function() {
-        var frame = element.find('.frame_container');
-        frame.html(frame.html());
-      };
+      scope.refreshFrame = regenFrame;
```

Comment:
Why not put the function directly on the scope instead of having it as a separate name that's then assigned here?